docs/developer-docs/integrations/sns/tokenomics/preparation.md

```diff
@@ -8,68 +8,42 @@ for examples using the
 [SNS tokenomics training & tools](https://wiki.internetcomputer.org/wiki/How-To:_SNS_tokenomics_configuration)
 as described in [this section](./tokenomics-intro.md).
 After deciding on these configurations, the next step is to
-translate them into 
-[initial parameters](../launching/launch-steps.md/#SNS-launch-step-preparation) <!-- NEW: UPDATE LINK --> that the SNS will be initialized with during the SNS launch.
+translate them into initial parameters that the SNS will be initialized 
+as the [first step of the SNS launch process](../launching/launch-steps-1proposal.md).
 
 
 This section explains on a technical, but high level, how initial parameters are set and how they affect different canisters.
 The [SNS CLI tool](https://github.com/dfinity/ic/tree/master/rs/sns/cli)
 allows you to set these parameters and includes detailed descriptions of all parameters.
 
 ### Initial parameters
-<!-- NEW: All SNS parameters are set in a **.yaml** file that can then be passed as an argument to the NNS proposal that triggers the SNS launch, both for testing and in production.
-There are some parameters that have to be set and others that are set to a default value but that can also be changed. To make sure that all parameters are set to valid values, that are also consistent
-with each other, there is a tool to validate a given input file. -->
-Apart from the decentralization swap parameters
-(see next section), all SNS parameters are set in a **.yaml** file that can 
-then be passed as an argument
-when installing the SNS, both for testing and in production.
-There are some parameters that have to be set and others that are set to a
-default value but that can also be changed.
-To make sure that all parameters are set to valid values, that are also consistent
-with each other, there is a tool to validate this input file. 
+All SNS parameters are set in a **.yaml** file that can then be passed as an argument to the NNS proposal that triggers the SNS launch, both for testing and in production. 
+To make sure that all parameters make sense and are also consistent with each other, it is recommended to test the proposal locally.
 
 **To create the SNS parameter yaml file and validate it, follow the steps
 ['Creating the configuration file'](https://github.com/dfinity/ic/tree/master/rs/sns/cli#creating-the-configuration-file)
 in the SNS CLI tool.
-The tool is also available through the dfx cache like this: `$(dfx cache show)/sns`**
+The tool is also available through the dfx cache like this: `$(dfx cache show)/sns` **
```

Comment:
**To create the SNS parameter yaml file, copy the template file found [here](https://github.com/dfinity/ic/blob/master/rs/sns/cli/sns_init_template.yaml) and fill in the fields with the desired values. An example configuration file with filled out fields can be found in [sns-testing](https://github.com/dfinity/sns-testing/blob/main/example_sns_init.yaml).**